zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/integration/ElasticsearchStorageExtension.java

```diff
@@ -71,6 +87,7 @@
 
     try {
       tryToInitializeSession();
+      checkForDeprecatedFeatures();
```

Comment:
if this is about index templates, possibly we can make this a test instead. ex we don't have ITEnsureIndexTemplate like in cassandra ITEnsureSchema. My assumption is that this is independent of api calls and so only needs to run once, but it should run after we setup the templates. having it as a separate test file will make this part more explicit and close to what fails imho.

Reply:
makes sense, moved to a different test